package.json

```diff
@@ -67,7 +67,8 @@
     "@seamapi/http": "0.25.3",
     "@seamapi/types": "1.153.2",
     "@seamapi/webhook": "1.0.0-rc.0",
-    "seamapi-types": "1.42.0"
+    "seamapi-types": "1.42.0",
+    "zod": "^3.21.4"
```

Comment:
This needs to be added a dependency or types will not work for consumers who use package managers that do not automatically install peer dependencies.

Reply:
This is for types only right? We're not actually including Zod in our bundle (it's a pretty large package)

Reply:
Yes it's only for types. This repo doesn't have a bundle that bundles external dependencies, it's only distributed as an esm module and a commonjs module.

Note that zod is already an optional peer dependency of `@seamapi/types` and is already being installed by npm users. I'm unaware of any alternative solution for dealing with "TypeScript  dependencies" like this without compromising the simplicity of just `yarn install seam`

Reply:
I wouldn't consider Zod too large:

> Tiny: 8kb minified + zipped
> https://zod.dev/?id=introduction